src/chains/ethereum/ethereum/src/api.ts

```diff
@@ -3426,39 +3426,83 @@ export default class EthereumApi implements Api {
    */
   @assertArgLength(0)
   async txpool_content(): Promise<Ethereum.Pool.Content<"private">> {
-    const { transactions, common } = this.#blockchain;
+    const { transactions } = this.#blockchain;
     const {
-      transactionPool: { executables, origins }
+      transactionPool: { executables, origins, processMap }
     } = transactions;
 
-    const processMap = (map: Map<string, Heap<TypedTransaction>>) => {
-      let res: Record<
-        string,
-        Record<string, Ethereum.Pool.Transaction<"private">>
-      > = {};
-      for (let [_, { array, length }] of map) {
-        for (let i = 0; i < length; ++i) {
-          const transaction = array[i];
-          const from = transaction.from.toString();
-          if (res[from] === undefined) {
-            res[from] = {};
-          }
-          // The nonce keys are actual decimal numbers (as strings) and not
-          // hex literals (based on what geth returns).
-          const nonce = transaction.nonce.toBigInt().toString();
-          res[from][nonce] = transaction.toJSON(
-            common
-          ) as Ethereum.Pool.Transaction<"private">;
-        }
-      }
-      return res;
-    };
-
     return {
       pending: processMap(executables.pending),
       queued: processMap(origins)
     };
   }
 
+
+  /**
+   * Returns the number of transactions created by specified address currently pending for inclusion in the next block(s), as well as the ones that are being scheduled for future execution only.
+   * 
+   * @param address - The account address
+   * @returns The transactions currently pending or queued in the transaction pool by address.
+   * @example
+   * ```javascript
```

Comment:
For some reason these examples aren't working in the generated docs - but the `txpool_content` example isn't working either. I can look into this!

Reply:
I found it!
We'll need to add this to the top of  all three txpool examples to keep the transactions from being mined so they actually stay in the pool:
```TypeScript
await provider.send("miner_stop")
```

Also, for the `txpool_contentFrom` example can we send _another_ transaction from a _different_ address to showcase the fact that it filters by address?